src/django_bootstrap5/html.py

```diff
@@ -17,8 +21,24 @@ def render_link_tag(url):
     return render_tag("link", attrs=attrs, close=False)
 
 
+def has_prefix(name, prefixes):
+    """Return whether the name has one of the given prefixes."""
+    return any(name.startswith(prefix) for prefix in prefixes)
```

Comment:
I wonder if we should add `_` to the prefix to prevent mistakenly matching `dataattribute=`?

Reply:
OK then, we do it like this